src/Kaleidoscope-LEDEffect-FunctionalColor.h

```diff
@@ -604,7 +604,8 @@ class FunctionalColor : public LEDMode {
   static FunctionalColor *lastFC;
   void onActivate(void) final;
   void update(void) final;
-  void refreshAt(byte r, byte c) final;
+  using LEDMode::refreshAt;
```

Comment:
This `using` declaration is only necessary until the deprecated compatibility functions in class `LEDMode` are removed (in some months).